pkg/pods/pod.go

```diff
@@ -421,6 +431,35 @@ func (pod *Pod) Uninstall() error {
 		return err
 	}
 
+	err = pod.pruneDockerResources()
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (pod *Pod) pruneDockerResources() error {
```

Comment:
how much trouble do you think it would be to reorganize all of this so it's in `pkg/docker`? probably that means changing the `artifact.Downloader` interface or creating a new interface on top, but probably we shouldn't keep piling docker stuff in this file